### ISSUE TITLE
Adding functionality to neo4j

### ISSUE THREAD

**Issue by PhantomScythe1738** [validas/NLX#3]
Hello Sir.
I want to add/create a graph where each node is basically the
word(and it has attribute word type-noun/verb/..), and nodes are connected
to each node will have maximum 2 connections(Let's take the example of 'The
weather is nice today.'
So as I have 5 words I will have 5 nodes, 'the' connected to 'weather',
'weather' connected to 'is' and so on. Each node will have property 'word
type'. This is wish to do for each line in the ('.nlx') file, so that I get something like a language model.

Sir, could you please guide me as to which files have to be changed for
being able to do this. (I still wish to have the neo4j graph dictionary,
this would be another graph)

**Comment by freshNfunky** [comment]
it's already implemented.
Just type in the sentence in the editor and it will resolve:
![Screenshot 2022-07-09 101200](https://user-images.githubusercontent.com/1439992/178097710-ddb5f76a-65c1-427d-a2b9-932055045159.png)

words are assigned to word types.
what you are looking after is the node type "GrammarCluster" in the database. Grammar clusters define which wrod types can create hierarchies upon each other to resolve the grammar tree.
![Screenshot 2022-07-09 102128](https://user-images.githubusercontent.com/1439992/178097883-692a6810-ce74-4a83-a7be-7e50d3f1ffaf.png)

**Comment by PhantomScythe1738** [comment]
Yes sir, but I wish to have a graph with the words(if in a .NLX file there
are 200 words, I wish to have 200 nodes(nodes being the words itself) and
each node will have property 'word type' with link to next word with
relation 'followed_by'. In the current model, the words do not seem to be
pushed, only the pattern is visible.

**Comment by freshNfunky** [comment]
The sentence itself is in the DOM-Tree locally,
but the words are in the DB as well, assigned to [1..*] Types.
If a Word is unknown to the DB then it's underlined with a blue jig-jag line. Then YOU define what it is in that context.
EG.: 
- "The _use_ of the software..." use here is a Noun
- "We _use_ soap to clean..." use here is a Verb
according to the Context and the Training it is either detected as Noun or as Verb.

it's not required to store a sentence in the DB but just the words and their relationship to other words and the context.
Just enter some Queries in the DB-Browser to explore the relationships
